### DISCUSSION TITLE
Guides on how to get new badges

### DISCUSSION THREAD

**Discussion by devkennyy** [community/community#27967] [Other Feature Feedback, Questions, & Ideas]
If anyone needs a guide or wants to earn the new badges, I made a playground for people to test and earn them. 
Check it out: [devkennyy/achivement-playground](https://github.com/devkennyy/achievement-playground)

**Comment by yusuftemel** [comment]
To achieve the 'Galaxy Brain' achievement, take note of the following:
Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.
Hope this answer has helped you, and all the best in getting the achievement! :)  [devkennyy](https://github.com/devkennyy)
https://github.com/Schweinepriester/github-profile-achievements

**Comment by anhvin007** [comment]
full list of achievement badges:
https://github.com/Schweinepriester/github-profile-achievements#achievements

**Comment by SharfShiplo** [comment]
After answering two discussions, You will get Galaxy Brain badge

More details here:
https://github.com/Schweinepriester/github-profile-achievements

**Comment by CodewithSegNet** [comment]
Work on some open-source projects.

1. Try searching for the technology you know For Example: React open-source projects

2. Interact with community discussions

**Comment by denverharry** [comment]
I also want to get all badges of github. I want to update my website [yes or no wheel](http://yesornowheel.org/) hosted on wordpress.

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by navodlakshan** [comment]
You can get a complete idea about getting GitHub Achievements by following the link below.
Link: https://github.com/drknzz/GitHub-Achievements?tab=readme-ov-file
